app/controllers/timelineable.rb

```diff
@@ -6,13 +6,13 @@ module Timelineable
   end
 
   def timeline
-    @display_date = display_date_as_time.to_date
+    @display_date = display_date_as_time
```

Comment:
Fair. Do you have a better idea for a name? It's basically the current time or midnight at the beginning of the specified date.